src/net/sourceforge/kolmafia/textui/parsetree/AggregateLiteral.java

```diff
@@ -62,6 +62,11 @@ public Value remove( final Value key, final AshRuntime interpreter )
 		return null;
 	}
 
+	@Override
+	public void clear()
+	{
+	}
```

Comment:
(Some of these invalid accesses might make sense as thrown exceptions, but this is fine either way.)